src/Swift.Runtime/src/Swift/SwiftOptional.cs

```diff
@@ -71,6 +71,18 @@ IntPtr ISwiftObject.MarshalToSwift(IntPtr swiftDest)
         return swiftDest;
     }
 
+    /// <summary>
+    /// Gets the protocol conformance descriptor for the given type
+    /// </summary>
+    /// <typeparam name="TProtocol"></typeparam>
+    /// <returns></returns>
+    static ProtocolConformanceDescriptor ISwiftObject.GetProtocolConformanceDescriptor<TProtocol>()
+        where TProtocol : class
+    {
+        // TODO: Implement
```

Comment:
```suggestion
        // TODO: https://github.com/dotnet/runtimelab/issues/2957
```

Please update the tracking issue if needed.

Reply:
https://github.com/dotnet/runtimelab/issues/2963